hmt_escrow/eth_bridge.py

```diff
@@ -232,6 +238,46 @@ def deploy_factory(gas: int = GAS_LIMIT, **credentials) -> str:
     return contract_addr
 
 
+def get_pk_from_address(wallet_addr: str) -> bytes:
+    """
+    Given a wallet address, uses the kvstore to pull down the public key for a user
+    in the hmt universe.  Requires that the `GAS_PAYER` environment variable be set to the
+    address that will be paying for the transaction on the ethereum network
+
+    Args:
+        wallet_addr (string): address to get the public key of
+
+    Returns:
+        bytes: the public key in bytes form
+
+    >>> import os
+    >>> from web3 import Web3
+    >>> get_pk_from_address('blah')
+    Traceback (most recent call last):
+    File "<stdin>", line 1, in <module>
+    File "/work/hmt_escrow/eth_bridge.py", line 255, in get_pk_from_address
+        raise ValueError('environment variable GAS_PAYER required')
+    ValueError: environment variable GAS_PAYER required
+    >>> os.environ['GAS_PAYER'] = Web3.toChecksumAddress('0x3895d913a9a231d2b215f402c528511b569c676d')
+    >>> get_pk_from_address(Web3.toChecksumAddress('0xc23546d9c10e85322cb63112ec034090b407a019'))
```

Comment:
This doesn't really test that we fetched the correct public key.

We should probably add a "set" method as well so we can test that function actually gives us the correct public key.

Also would rename it from `get_pk_from_address` to `get_pub_key_from_addr`.

Reply:
Ya I can do those renames .. Also this was from the other day when the tests didn't work, fixing them up now.  Was just trying to figure out how to do a test set of a kv pair, but I don't think testing that or the function should live in this repo .. we should use the api of kvstore

Reply:
Well our kvstore returns a string when we do the get. `get_pub_key_from_addr` converts it to bytes so we add a bit functionality there which would be good to test. Can't count the amount of times and hours wasted because I was expecting bytes or str and vice versa.